calico_node/startup/startup.go

```diff
@@ -43,6 +43,9 @@ const (
 // For testing purposes we define an exit function that we can override.
 var exitFunction = os.Exit
 
+// Prefix all logs that we output to in `calicotl node run` with a specific string.
+var calicoctlLogPrefix = "progress: "
```

Comment:
Hm.  Can we decide which logs to send to screen using log level perhaps? INFO and above get sent to screen via calicoctl?  Would be easy to then say, send DEBUG logs to screen for a `--verbose` option or something.

I'd also like to remove references to `calicoctl` within the `calico/node` image to avoid tying the two together unnecessarily.  At a minimum can we rename this variable to something else like `progressLogPrefix`?